.cbmc-batch/jobs/aws_cryptosdk_cmm_base_init/aws_cryptosdk_cmm_base_init_harness.c

```diff
@@ -23,10 +23,17 @@
 #include <proof_helpers/utils.h>
 
 void aws_cryptosdk_cmm_base_init_harness() {
+    /* Nondet input */
     struct aws_cryptosdk_cmm cmm;
     const struct aws_cryptosdk_cmm_vt vtable;
+
+    /* Assumptions */
     *(char **)(&vtable.name) = ensure_c_str_is_allocated(SIZE_MAX);
```

Comment:
why is this not part of `ensure_vtable_has_allocated_members`

Reply:
It is, updated to use this function!